pkg/agent/variables.go

```diff
@@ -191,8 +202,14 @@ func getOutBoundCmd(nbc *datamodel.NodeBootstrappingConfiguration, cloudSpecConf
 
 	// curl on Ubuntu 16.04 (shipped prior to AKS 1.18) doesn't support proxy TLS.
 	// so we need to use nc for the connectivity check.
-	clusterVersion, _ := semver.Make(cs.Properties.OrchestratorProfile.OrchestratorVersion)
-	minVersion, _ := semver.Make("1.18.0")
+	clusterVersion, err := semver.Make(cs.Properties.OrchestratorProfile.OrchestratorVersion)
+	if err != nil {
+		return err.Error()
+	}
+	minVersion, err := semver.Make("1.18.0")
+	if err != nil {
+		return err.Error()
```

Comment:
the return value here is really used during node provisioning, you can't return errors here

Reply:
I see, the linter was basically upset that the err wasn't checked. Would it be better to return an empty string?